content/docs/setup/kubernetes/multicluster-install/vpn/index.md

```diff
@@ -99,54 +114,72 @@ Proceed to one of the options for connecting the remote cluster to the local clu
     --set global.remoteZipkinAddress=${ZIPKIN_POD_IP} > $HOME/istio-remote.yaml
     {{< /text >}}
 
-1.  Create a namespace for remote Istio.
+1.  Create an `istio-system` namespace for remote Istio with the following
+    command:
 
     {{< text bash >}}
     $ kubectl create ns istio-system
     {{< /text >}}
 
-1.  Instantiate the remote cluster's connection to the Istio control plane:
+    {{ <info_icon> }} All clusters must have the same namespace for the Istio components. It is possible to override the "istio-system" name on the main cluster as long as the namespace is the same for all Istio components in all clusters.
```

Comment:
This needs to be {{< info_icon >}} (i.e. no spaces between the {{ and the <, and between the > and }}). As it is, it's not rendering correctly.

Reply:
Done